### ISSUE TITLE
hugging face show some internal error while runnig  feature extraction example in hugging face

### ISSUE THREAD

**Issue by huzaifansari54** [shamblett/huggingface_client#11]
final input = 'A string of text from which to extract the features.';
      final params = ApiQueryNLPFeatureExtraction(
          inputs: [input], options: InferenceOptions(waitForModel: true));
      final result = await apiInstance.queryNLPFeatureExtraction(
          taskParameters: params,
          model: 'sentence-transformers/paraphrase-xlm-r-multilingual-v1');
      if (result!.isNotEmpty) {
        for (final row in result) {
          print(row?.representationFeatures);
        }
      } else {
        print('Inference task API Feature Extraction returned empty result');
      }

**while runnig this code i get  this error**
ApiException 400: {"error":["Input should be a valid dictionary or instance of SentenceSimilarityInputsCheck: received `['A string of text from which to extract the features.']` in `parameters`"]})

i think hugging face api changes their parameters
I also checked  the text generation api it is also not work, could you suggest how to solve this?

**Comment by shamblett** [comment]
Looks as though Hugging face have changed their parameters as you suggest, this is a bit naughty as it causes breakage for everybody, they should really version their interfaces to stop this. s far as I know this is only solvable by a package update.

**Comment by huzaifansari54** [comment]
can u suggest me how to solve this issue?

**Comment by shamblett** [comment]
Yes, as I've said above it needs a code change in the package to fix this. Feel free to raise a pull request if you wish.

**Comment by huzaifansari54** [comment]
I get this error while getting a response from testToImage api the response content type is image/jpeg i dont know how to handle this issue in dio pkg i know but in HTTP dart I don't know kindly help me how to solve it 

**EXCEPTION**
Exception when calling Inference task API Text  To Image : FormatException: Unexpected character (at character 1)
ÿØÿàJFIFÿÛC		
^
 - exitin
 
**Response**
body : "ÿØÿàJFIFÿÛC		 $.' ",#(7),01444'9=82<.342ÿÛC			2!!22222222222222222222…"
"content-type" -> "image/jpeg"

**Comment by huzaifansari54** [comment]
i solve  it 👍🏿👍🏿👍🏿👍🏿

**Comment by huzaifansari54** [comment]
i solve it actually i dont know but we use  wrong model thats why we get wrong parameters error
use this mode facebook/bart-base 
i thing we should givepre define list of models so devloper use it like for image to text we give list o working models
what you things

**Comment by shamblett** [comment]
Great thanks, I've applied a few very minor edits but otherwise fine. Package re released at version 1.3.0
